mobile/src/calendar/calendar-page-weekday.tsx

```diff
@@ -85,11 +87,31 @@ export class WeekDayCircle extends Component<WeekDayCircleProps> {
 
     //----------------------------------------------------------------------------
     private isSelectedDay(day: DayModel) {
-        return (
-            this.props.selectedDay &&
-            this.props.selectedDay.date &&
-            this.props.selectedDay.date.isSame(day.date, 'day')
-        );
+        const { selectedDay } = this.props;
+
+        return selectedDay && selectedDay.date && selectedDay.date.isSame(day.date, 'day');
+    }
+
+    //----------------------------------------------------------------------------
+    private getColorFor(day: DayModel) {
+        const { customTextColor } = this.props;
+
+        if (day.belongsToCurrentMonth) {
+
+            if (this.isSelectedDay(day)) {
+                return Style.color.white;
+            }
+
+            if (business.isWeekendDay(day.date)) {
+                return Style.color.gray;
+            }
+
+            if (customTextColor) {
```

Comment:
Why check for custom text color is on the last place? Shouldn't custom color win?

Reply:
This variable really has confusing name and actually wasn't used. I've removed in e560f680693caf5e9822d20e26502b19a80f0581